Azure functions are not supported in monitors yet (#2909)
SumoLogic/sumologic-documentation: docs/alerts/webhook-connections/microsoft-azure-functions.md

description: You can trigger an Azure Function directly from a Sumo Logic alert by configuring a webhook connection.
---

> You can trigger an Azure Function directly from a Sumo Logic alert by configuring a webhook connection. Once you set up the webhook connection, you'll have the option to use it in a [Scheduled Search](schedule-searches-webhook-connections.md). Azure functions are not supported in [Monitors](/docs/alerts/monitors) yet.

Reviewer: ```suggestion
You can trigger an Azure Function directly from a Sumo Logic alert by configuring a webhook connection. Once you set up the webhook connection, you can use it in a [Scheduled Search](schedule-searches-webhook-connections.md). Currently, Azure functions are not supported in [Monitors](/docs/alerts/monitors).
```
